.github/ISSUE_TEMPLATE/folders.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Translation for main folders
+about: Do you want to translate the main folders of the Wiki to your language?
+title: Translation for main folders
```

Comment:
I think we should add 2 label types and automatically add the right labels for them, just so it is easier to filter etc

Reply:
I added lables "move" and "translation". The latter one is meh, but I don't know any better one.